docs/vnext/openapi.yaml

```diff
@@ -308,22 +227,22 @@ components:
     HeaderTraceParent:
       name: traceparent
       required: true
-      description: Standard W3C trace context header generated by OpenTelemetry. Providers should use the header if they support OpenTelemetry.
+      description: Standard W3C trace context header generated by OpenTelemetry. Extensions should use the header if they support OpenTelemetry.
       in: header
       schema:
         type: string
     HeaderTraceState:
       name: tracestate
       required: true
-      description: Standard W3C trace context header generated by OpenTelemetry. Providers should use the header if they support OpenTelemetry.
+      description: Standard W3C trace context header generated by OpenTelemetry. Extensions should use the header if they support OpenTelemetry.
       in: header
       schema:
         type: string
-    ProviderVersion:
-      name: providerVersion
+    ExtensionVersion:
+      name: extensionVersion
       in: path
       required: true
-      description: Version of the provider. This is specified by the user with the provider import syntax in the Bicep or ARM template file. It is recommended for the extensibility provider to validate the request payload against the provider version, but it's not mandatory.
+      description: Version of the extension. This is specified by the user with the extension import syntax in the Bicep or ARM template file. It is recommended for the extension service to validate the request payload against the extension version, but it's not mandatory.
       schema:
         type: string
     ReferenceId:

```

Comment:
Should this be deleted? It seems we aren't using ReferenceId anymore

Reply:
Yeah good find. Will remove it.